src/components/Shortcuts/ShortcutsView.spec.jsx

```diff
@@ -5,18 +5,18 @@ import '@testing-library/jest-dom'
 
 import { ShortcutsView } from './ShortcutsView'
 import AppLike from 'test/AppLike'
-import MuiCozyTheme from 'cozy-ui/transpiled/react/MuiCozyTheme'
+import CozyTheme from 'cozy-ui/transpiled/react/providers/CozyTheme'
 
 describe('Shortcuts', () => {
   it('Should display nothing if nothing was found', () => {
     const { container } = render(
       <AppLike>
-        <MuiCozyTheme>
+        <CozyTheme>
           <ShortcutsView shortcutsDirectories={null} />
-        </MuiCozyTheme>
+        </CozyTheme>
       </AppLike>
     )
-    expect(container).toBeEmptyDOMElement()
+    expect(container).toMatchSnapshot()
```

Comment:
Why did you prefer `toMatchSnapshot` instead `toBeEmptyDOMElement` ?

Reply:
because now this is not an empty dom element. We have the `CozyTheme` so we have a `div` created

Reply:
I hadn't thought of that